File: python_draft/src/cpp/type1/calculate_average_comments.cpp
#include <vector>
// Computes mean value
double calculateAverage1b(const std::vector<int>& numbers){ 
    long long total = 0; // Sum all elements
    for(int n : numbers) total += n;
    // Count elements
    return (double) total / numbers.size(); // Return average
}

File: python_draft/src/java/type1/CalculateAverageComments.java
public class CalculateAverageComments {
// Type I variant 2 - comment addition/removal
public static double calculateAverage1b(int[] numbers) { // Computes mean value
    int total = 0; // Sum all elements
    for (int n : numbers) total += n;
    int count = numbers.length; // Count elements
    return (double) total / count; // Return average
}

}
